### PR TITLE
feat : single product page

### PR DESCRIPTION
**Pull request by anuj-kumary**
Added Product details page
Preview : https://deploy-preview-13--eaglestores-react.netlify.app/

### CODE REVIEW

File: eagle-stores/src/App.js [hunk 1]
```diff
@@ -47,6 +48,7 @@ function App() {
         <Route path='/login' element={<Login />} />
         <Route path='/signup' element={<Signup />} />
         <Route path='/profile' element={<Profile />} />
+        <Route path='/product/:productId' element={<ProductDetails />} />
```
Comment (PeeyushPrashant) [inline eagle-stores/src/App.js:51]: good use of React Router

**Review by PeeyushPrashant — APPROVED** [review]
code looks modular.

Suggestion- It will be better if you provide "cursor:pointer" on cards on products page

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge ✅

File: eagle-stores/src/App.js (hunk 1 above)
Comment (anuj-kumary) [reply eagle-stores/src/App.js:51]: Okay I'll add that

**Review by hrishi82 — APPROVED** [review]
code looks good to me, can be merged!

**Review by Rohanmond — APPROVED** [review]

**Pull request merged**